lib/geoblacklight/references.rb

```diff
@@ -72,9 +73,9 @@ def download_types
     ##
     # Returns all of the Esri webservices for given set of references
     def esri_webservices
-      %w[tiled_map_layer dynamic_map_layer feature_layer image_map_layer].map do |layer_type|
+      %w[tiled_map_layer dynamic_map_layer feature_layer image_map_layer].filter_map do |layer_type|
```

Comment:
comment: I like `filter_map`! Much better than the `compact` at the end.

Reply:
I can't take credit for this because I am pretty sure this was a rubocop automatic fix, but good to know it is better!